### PR TITLE
Update guide-import-zip.md

### PR DESCRIPTION
**Pull request by AlexandreHT**
Update needed because:
- UI for offline import have changed (Pull button)
- I detailed the different ways to run abapGit depending on the flavour of the abapGit installed
- I followed guidelines found in https://developers.google.com/tech-writing (which is an amazing (and short) open class on technical writing. I believe that large screenshot tend to confuse a step by step procedure (+ UI tends to change quicker than doc). Yet if you feel like screenshot are needed I can add those.

### CODE REVIEW

**Comment by AlexandreHT** [comment]
It looks like the same but it's not : 
* In installation, you have to run standalone version to install developper version. 
* In offline zip, you may run standalone version or developper version

An alternative would be to replace step-by-step procedure in guide-install.html by a link to this page. I thought about some of my colleagues who are not familiar with abapGit and concluded it would be clearer that way.

Your call anyway :)

File: docs/guide-import-zip.md [hunk 1]
```diff
@@ -4,20 +4,24 @@ category: offline projects
 order: 10
 ---
 
-1: Run report ZABAPGIT via SE38 or SA38
+## Download zip archive ## 
+Offline projet is based on a zip archive that contains all the git files.
```
Comment (FreHu) [inline docs/guide-import-zip.md:8]:
```suggestion
An offline project is based on a zip archive that contains all the git files.
```

File: docs/guide-import-zip.md [hunk 2]
```diff
@@ -4,20 +4,24 @@ category: offline projects
...
 
-2: Click "+ Offline"
-![](img/offline1.png)
+First download latest version of the projet you want to import.
```
Comment (FreHu) [inline docs/guide-import-zip.md:10]:
```suggestion
First, download the latest version of the project you want to import. 
```

File: docs/guide-import-zip.md [hunk 3]
```diff
@@ -4,20 +4,24 @@ category: offline projects
...
+Offline projet is based on a zip archive that contains all the git files.
 
-2: Click "+ Offline"
-![](img/offline1.png)
+First download latest version of the projet you want to import. 
 
-3: Enter project name eg. "abapOpenChecks", and an empty package eg. "$AOC"
-![](img/offline2.png)
+* For GitHub :
+   1. Go to the projet main page (eg. https://github.com/larshp/abapOpenChecks)
```
Comment (FreHu) [inline docs/guide-import-zip.md:13]:
```suggestion
   1. Go to the project main page (e.g. https://github.com/larshp/abapOpenChecks)
```

File: docs/guide-import-zip.md [hunk 4]
```diff
@@ -4,20 +4,24 @@ category: offline projects
...
-![](img/download_zip.png)
-
-5: Click "Import ZIP" in abapGit
-![](img/offline3.png)
-
-6: Choose file downloaded in step 4
+## Import zip archive ##
+1. Connect to SAP in language **EN**
+2. Run abapGit:
+  * You have installed standalone version : in transaction `SA38`, run the program you created  (`ZABAPGIT_STANDALONE` or `ZABAPGIT_FULL` or other)
```
Comment (FreHu) [inline docs/guide-import-zip.md:20]:
```suggestion
  * If you have installed the standalone version : in transaction `SE38`, run the program you created  (`ZABAPGIT_STANDALONE` or `ZABAPGIT_FULL` or other)
```

File: docs/guide-import-zip.md [hunk 5]
```diff
@@ -4,20 +4,24 @@ category: offline projects
...
+  * You have installed developper version : call transaction `ZABAPGIT`
```
Comment (FreHu) [inline docs/guide-import-zip.md:21]:
```suggestion
  * You have installed the developer version : run transaction `ZABAPGIT`
```

File: docs/guide-import-zip.md [hunk 6]
```diff
@@ -4,20 +4,24 @@ category: offline projects
...
+3. Select **+ Offline**
+4. Enter projet name and select existing package or create new package
```
Comment (FreHu) [inline docs/guide-import-zip.md:23]:
```suggestion
4. Enter the project name and select an existing package or create a new package
```

**Comment by larshp** [comment]
thanks 👍

**Pull request merged**